### PR TITLE
Ledger API: scale all timeLimits in tests with ScaledTimeSpans

### PR DESCRIPTION
**Pull request by gleber-da**
This ensures that more tests will be automatically scaled with `--timeout-scale-factor` command line option.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/tools/ledger-api-test-tool/index.rst [hunk 1]
```diff
@@ -160,3 +160,19 @@ will still print information about failed tests.
 
 We used this flag during tool development to ensure that the tool does not
 always return success.
+
+Tuning the testing behaviour of the tool
+========================================
+
+Use command line options ``--timeout-scale-factor`` and ``--command-submission-ttl-scale-factor`` to tune timeouts applied by the tool:
```
Comment (gerolf-da) [inline docs/source/tools/ledger-api-test-tool/index.rst]:
```suggestion
Use the command line options ``--timeout-scale-factor`` and ``--command-submission-ttl-scale-factor`` to tune timeouts applied by the tool:
```

File: docs/source/tools/ledger-api-test-tool/index.rst [hunk 2]
```diff
@@ -160,3 +160,19 @@ will still print information about failed tests.
...
+
+- set ``--timeout-scale-factor`` to floating point value higher than 1.0 to make
```
Comment (gerolf-da) [inline docs/source/tools/ledger-api-test-tool/index.rst]:
```suggestion
- set ``--timeout-scale-factor`` to a floating point value higher than 1.0 to make
```

File: docs/source/tools/ledger-api-test-tool/index.rst [hunk 3]
```diff
@@ -160,3 +160,19 @@ will still print information about failed tests.
...
+  the tool wait longer for expected events coming from the DAML ledger
+  implementation under test, conversely use values smaller than 1.0 to make it
+  wait shorter;
+- set ``--command-submission-ttl-scale-factor`` to value higher than 1.0 to make
```
Comment (gerolf-da) [inline docs/source/tools/ledger-api-test-tool/index.rst]:
```suggestion
- set ``--command-submission-ttl-scale-factor`` to a value higher than 1.0 to make
```

File: docs/source/tools/ledger-api-test-tool/index.rst [hunk 4]
```diff
@@ -160,3 +160,19 @@ will still print information about failed tests.
...
+  it generate Ledger API Commands with higher than default Max Record Time,
```
Comment (gerolf-da) [inline docs/source/tools/ledger-api-test-tool/index.rst]:
```suggestion
  the test tool generate Ledger API Commands with higher than default maximum record time,

**Comment by gleber-da** [comment]
@gerolf-da PTAL

File: docs/source/tools/ledger-api-test-tool/index.rst [hunk 5]
```diff
@@ -160,3 +160,19 @@ will still print information about failed tests.
...
+  which might be necessary for DAML ledger implementations which takes long time
```
Comment (gerolf-da) [inline docs/source/tools/ledger-api-test-tool/index.rst]:
```suggestion
  which might be necessary for DAML ledger implementations which take a long time
```

File: docs/source/tools/ledger-api-test-tool/index.rst [hunk 6]
```diff
@@ -160,3 +160,19 @@ will still print information about failed tests.
...
 always return success.
+
+Tuning the testing behaviour of the tool
+========================================
+
+Use command line options ``--timeout-scale-factor`` and ``--command-submission-ttl-scale-factor`` to tune timeouts applied by the tool:
+
+- set ``--timeout-scale-factor`` to floating point value higher than 1.0 to make
+  the tool wait longer for expected events coming from the DAML ledger
+  implementation under test, conversely use values smaller than 1.0 to make it
```
Comment (gerolf-da) [inline docs/source/tools/ledger-api-test-tool/index.rst]:
```suggestion
  implementation under test. Conversely use values smaller than 1.0 to make it
```

File: docs/source/tools/ledger-api-test-tool/index.rst [hunk 7]
```diff
@@ -160,3 +160,19 @@ will still print information about failed tests.
...
+  wait shorter;
+- set ``--command-submission-ttl-scale-factor`` to value higher than 1.0 to make
+  it generate Ledger API Commands with higher than default Max Record Time,
+  which might be necessary for DAML ledger implementations which takes long time
+  to commit a proposed transaction, conversely use values smaller than 1.0 to
```
Comment (gerolf-da) [inline docs/source/tools/ledger-api-test-tool/index.rst]:
```suggestion
  to commit a proposed transaction. Conversely use values smaller than 1.0 to
```

Comment (gleber-da) [reply docs/source/tools/ledger-api-test-tool/index.rst]: Done

File: docs/source/tools/ledger-api-test-tool/index.rst (hunk 6 above)
Comment (gleber-da) [reply docs/source/tools/ledger-api-test-tool/index.rst]: Done

File: docs/source/tools/ledger-api-test-tool/index.rst (hunk 5 above)
Comment (gleber-da) [reply docs/source/tools/ledger-api-test-tool/index.rst]: Done

File: docs/source/tools/ledger-api-test-tool/index.rst (hunk 4 above)
Comment (gleber-da) [reply docs/source/tools/ledger-api-test-tool/index.rst]: Done

File: docs/source/tools/ledger-api-test-tool/index.rst (hunk 3 above)
Comment (gleber-da) [reply docs/source/tools/ledger-api-test-tool/index.rst]: Done

File: docs/source/tools/ledger-api-test-tool/index.rst (hunk 2 above)
Comment (gleber-da) [reply docs/source/tools/ledger-api-test-tool/index.rst]: Done

File: docs/source/tools/ledger-api-test-tool/index.rst (hunk 1 above)
Comment (gleber-da) [reply docs/source/tools/ledger-api-test-tool/index.rst]: Done

**Comment by gleber-da** [comment]
@gerolf-da All done

File: docs/source/tools/ledger-api-test-tool/index.rst [hunk 8]
```diff
@@ -160,3 +160,20 @@ will still print information about failed tests.
...
+Tuning the testing behaviour of the tool
+========================================
+
+Use the command line options ``--timeout-scale-factor`` and
+``--command-submission-ttl-scale-factor`` to tune timeouts applied by the tool:
+
+- set ``--timeout-scale-factor`` to a floating point value higher than 1.0 to make
+  the tool wait longer for expected events coming from the DAML ledger
+  implementation under test. Conversely use values smaller than 1.0 to make it
+  wait shorter;
```
Comment (gerolf-da) [inline docs/source/tools/ledger-api-test-tool/index.rst]:
```suggestion
  wait shorter.
```

Sorry, missed this the first time around.

Comment (gleber-da) [reply docs/source/tools/ledger-api-test-tool/index.rst]: I've also updated the capitalization of the first word of the sentences in the bullet points.

Comment (gerolf-da) [reply docs/source/tools/ledger-api-test-tool/index.rst]: Thanks.

**Pull request merged**